src/main/java/com/vmware/vipclient/i18n/base/cache/MessageCacheItem.java

```diff
@@ -73,18 +73,8 @@ public synchronized boolean isExpired() {
     	if (VIPCfg.getInstance().getCacheExpiredTime() != 0) {
     		return false;
     	}
-    	
-    	Long responseTimeStamp = this.getTimestamp();
-    	if (responseTimeStamp == null) {
-    		return true;
-    	}
-    	
-    	Long maxAgeResponse = this.getMaxAgeMillis();
-    	if (maxAgeResponse != null) {
-    		maxAgeMillis = maxAgeResponse;
-    	}
-    		  	
-    	return System.currentTimeMillis() - responseTimeStamp > maxAgeMillis;
+
+    	return System.currentTimeMillis() - this.getTimestamp() > this.getMaxAgeMillis();
```

Comment:
Same comment with xiaochao's comment to my PR:
1. change '>' to '>=';
2. add this line to synchronized block, or the timestamp and the maxAgeMillis may not match.

Reply:
Hi Huihui! The whole method is already synchronized.
I will change the '>' to '>=' in another PR because these quick clean up PRs must have no code logic change in them. https://github.com/vmware/singleton/issues/824

Reply:
1.  Yes, I have that listed for another PR. It's not related to this.
2.  The while method is already synchronized. Nonetheless, it is also not related to the PR.